index.html

```diff
@@ -1362,17 +1373,17 @@ <h3>Issuance Date</h3>
 
         <p>
 This specification defines the following <a>property</a> for expressing the
-date and time when the <a>credential</a> was issued:
+date and time when the <a>verifiable credential</a> was issued:
         </p>
 
         <dl>
           <dt><var>issuanceDate</var></dt>
           <dd>
-A <a>verifiable credential</a> MUST have an <code>issuanceDate</code> <a>property</a>.
-The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!RFC3339]] combined date and time string representing the date
-and time the <a>credential</a> was issued. Note that this date represents the
-earliest date when the information associated with the
+A <a>verifiable credential</a> MUST have an <code>issuanceDate</code>
+<a>property</a>. The value of the <code>issuanceDate</code> <a>property</a> MUST
+be a string value of an [[!RFC3339]] combined date and time string representing
+the date and time the <a>credential</a> was issued. Note that this date
```

Comment:
<a>credential</a> should be <a>verifiable credential</a>

Reply:
the rest of the changes look good to me.

Reply:
Fixed.